### PR TITLE
v2 beta announcement blog post

### PR DESCRIPTION
**Pull request by m-allanson**
Work in progress

### CODE REVIEW

File: docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The [v2 roadmap](https://github.com/gatsbyjs/gatsby/projects/2) details everything else we're planning for v2. Improvements include:
+
+- Additional build optimisations
+- Better error messaging
+- PostCSS configuration updates
+
+## Getting involved
+
+We ❤️ your feedback. Try out the [migration guide](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/) and let us know how you get on. [Issues](https://github.com/gatsbyjs/gatsby/issues) and [pull requests](https://github.com/gatsbyjs/gatsby/pulls) are welcome (for real).
```
Comment (avigoldman) [inline docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md]: Should it be `https://next.gatsbyjs.org/docs/migrating-from-v1-to-v2/`?

File: docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+tags: ["v2", "gatsby"]
+---
+
+We’re excited to announce that today we shipped the first beta for Gatsby v2! 35 contributors have made over 300 commits to v2 since Gatsby v1 was released in July 2017.
+
+## What's coming in v2?
+
+Gatsby v2 builds on the foundations of v1 to introduce a range of improvements:
+
+- Out With Layouts, In With StaticQuery
```
Comment (KyleAMathews) [inline docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md]: Let's say "Gatsby Layout components"  (or something equivalent) — "layouts" still exist in v2 — they're just your components now not a special things.

File: docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  Layouts were a common source of confusion in v1. After an RFC and [informative community discussion](https://github.com/gatsbyjs/rfcs/blob/master/text/0002-remove-special-layout-components.md), we decided to remove them and create `StaticQuery` which allows GraphQL queries to be added to any component. For more information on migrating your layouts, check out our Life After Layouts post (coming soon).
+
+- More consistent APIs
```
Comment (KyleAMathews) [inline docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md]: Maybe the header should be "Improve APIs" as only some of the changes were for consistency.

File: docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  To see the full list of renamings, take a look at [image queries](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#rename-responsive-image-queries), [actions](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#rename-code-classlanguage-textboundactioncreatorscode-to-code-classlanguage-textactionscode) and the [babel](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#change-code-classlanguage-textmodifybabelrccode-to-code-classlanguage-textoncreatebabelconfigcode) and [webpack](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#change-code-classlanguage-textmodifywebpackconfigcode-to-code-classlanguage-textoncreatewebpackconfigcode) API hooks.
+
+- Hotter Hot Reloading
+
+  Previously, hot reload had been dependent on new webpack builds. On larger sites, that could take ten or fifteen seconds -- more like lukewarm reloading, really. With [_ludicrous mode_](https://twitter.com/gatsbyjs/status/974507205121617920) we’ve decoupled webpack so that hot reloading is always instantaneous.
```
Comment (KyleAMathews) [inline docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md]: We need to distinguish between code hot reloading and data hot reloading — code hot reloading is obviously still going through webpack where this paragraph confuses the issue.

File: docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+- Upgraded Dependencies
+
+  These are the some of the least obvious but most important changes. Our dependency upgrades help us take advantage of hundreds of other engineers’ work across the incredible modern JS ecosystem.
```
Comment (KyleAMathews) [inline docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md]: I find "least obvious" a bit odd of phrasing — they are fairly obvious to many people. I'd introduce this by saying something like "Gatsby is built on top of amazing JavaScript libraries. We're delighted to ship the latest improvements from webpack, Babel, and React to you" — or something like that.

**Comment by m-allanson** [comment]
@KyleAMathews this has been updated

**Pull request merged**